cirq/circuits/text_diagram_drawer.py

```diff
@@ -253,7 +339,11 @@ def _pad_into_multiline(width: int,
         col_width = max(1, max(len(sub_row[col])
                                for row in multiline_grid
                                for sub_row in row))
-        col_width += horizontal_spacing
+        if groups_are_vertical:
+            col_width += horizontal_spacing
+        else:
+            col_width = _pad_border(border, col, col_width, horizontal_spacing,
```

Comment:
This makes the previous `col_width` assignment a dead write in this case. Move that previous assignment into the `if`'s positive case.

Reply:
Oops. Obsoleted by my huge design change comment.